src/DynamoCoreWpf/UI/SharedResourceDictionary.cs

```diff
@@ -64,6 +64,8 @@ public static class SharedDictionaryManager
         private static ResourceDictionary _connectorsDictionary;
         private static ResourceDictionary _portsDictionary;
```

Comment:
is this now inports only? Can we rename it?

Reply:
Ahh this one is deprecated essentially.  I will mark the public method as deprecated